cluster/juju/layers/kubernetes-worker/tests/00-setup

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
```

Comment:
This file can be eliminated by tests.yaml specifying packages. I would suggest we use that.